OpenInfraSummit2023/OpenOperations-One-Year-in.md

```diff
@@ -0,0 +1,38 @@
+# Open Operations - One Year in
+
+Authors: Kurt Garloff, Felix Kronlage-Dammers
+
+Track: TBD
+
+Technical Level: Beginners
+
+## Abstract
+
+Open Source has changed the way software is being developed and used over the
+last decades. The Open Infrastructure Community has been successful in evolving
+and defining good practices for the collaborative development with the Four
+Opens. The Open Operations movement is tackling a related problem: How can we
+effectively establish collaboration in operating complex IT technology?
+The authors have presented some ideas in the 2022 Berlin summit.
```

Comment:
"The Open Operations movement is building upon these ideas in order to extend them to the scope of  effectively establish collaboration in operating complex IT technology.
The authors presented the initial idea at the 2022 Berlin summit. Since then this has further evolved."

Reply:
Added this (with s/establish/establishing/).